Make FileUtils.mkdir_p act more like mkdir(1) -p, by not attempting
to create directories that already exist.

Index: lib/fileutils.rb
--- lib/fileutils.rb.orig
+++ lib/fileutils.rb
@@ -377,7 +377,7 @@ module FileUtils
       end
       stack.reverse_each do |dir|
         begin
-          fu_mkdir dir, mode
+          fu_mkdir dir, mode unless File.directory?(dir)
         rescue SystemCallError
           raise unless File.directory?(dir)
         end
